modules/node_modules/@ncigdc/routes/CartRoute.js

```diff
@@ -34,6 +35,15 @@ export default connect(state => (state.cart))(
       Container={CartPage}
       queryConfig={new CartRoute(props)}
       environment={Relay.Store}
+      render={({ error, props }) => {
```

Comment:
haha it's weird but it makes 100% sense if you look at the component. it just takes a value from 0 - 100 to figure out its width. if someone wants to play with it to make it go to the other side or listen to an onfinished event to do something, go for it. people forget that before the red bar there was nothing and it looked like the site was freezing constantly